### DISCUSSION TITLE
Opening file with line number (at launched editor)

### DISCUSSION THREAD

**Discussion by l4l** [helix-editor/helix#7009] [General]
Similar to #3277 but when the helix has already been launched.

Perhaps support in the file picker opening files with format `abc.xyz:123`?

Generally exact matches should be preferred anyway, e.g if both "abc.xyz:123" and "abc.xyz" exist then the first one should go earlier in the listing. Opening "abc.xyz" then moves cursor to line 123.

**Comment by pascalkuthe** [comment]
you can already use `:open <path>:LINE`. In the picker, this doesn't make any sense and is not supported in any fuzzy matching tools I am aware of. You can also easily type `<line>G` after you have opened a file which is just as fast as typing it in the picker.

**Reply by l4l** [reply]
Well, that somewhat works in sublime text:

[x.webm](https://github.com/helix-editor/helix/assets/5658339/79bf40ae-fd55-4214-962a-4b0d3509051f)

`:open` is a good option indeed, but it's not fuzzy and sometimes relative path doesn't in the current directory. Typing `<line>G` after opening basically forbids you from copying the whole line (e.g from stacktrace).

**Comment by jaydorsey** [comment]
Even a partial match on the file finder would be a nice feature, even if it didn't jump to the line number. Very frequently copy traces from debugging, etc. w/ a line number for specs so I can run the spec from the command line + I often need to open the file.

I'm not sure about libraries available to helix, but both the telescope picker in nvim supports this behavior, as does vs code (it seems to ignore the `:123` and finds the file, even if it doesn't jump to the line)
